Tests/SwiftDocCTests/Indexing/RenderIndexTests.swift

```diff
@@ -657,6 +657,11 @@ final class RenderIndexTests: XCTestCase {
                         "title" : "My Article",
                         "icon" : "plus.svg",
                         "type" : "article"
+                      },
```

Comment:
I think the test is now poorly named, since it tests more than CustomIcons at this point. In my mind there are two alternatives. Either we break off the tests into individual tests and instead of `XCTAssertEqual` on the full string, we assert on the presence of individual elements. alternatively we just rename this test to `testRenderIndexGenerationWithBookLikeContent` although I am not sure I like to refer to all of these as being book-like since I can see things like this new tab navigator being useful for large amounts of related screenshots in an article within conceptual documentation for an app or something.

Reply:
Ah I disagree. I think it's still only testing custom icons- if the navigator index doesn't include just regular article pages in it then something has gone _really_ wrong.

I just had to update the test to reflect the content of the DocC catalog since I added an additional page for a different test in `RenderNodeTranslatorTests` but the main thing this is testing for is still the special page that customizes an icon.

Reply:
I see what you mean, I think it's fine for this review then. FTR though I think that for someone new to the code base without the context of this PR, it is less clear what the test is checking for. We should come up with a better way of testing individual render index features without having to do this all the time.